syntax = "proto3";

// Not necessary for Python but should still be declared to avoid name collisions 
// in the Protocol Buffers namespace and non-Python languages
package protoblog;


message Struct {
    int32 PackageID = 1;
    int32 PersonID = 2;
    string Name = 3;
    map<string, int32> Inventory = 4;
    string CurrentLocation = 5;
}